templates/publisher/_repo-disconnect-modal.html

```diff
@@ -0,0 +1,26 @@
+<div class="p-modal" data-js="repo-disconnect-modal" id="repo-disconnect-modal" style="display: none;">
+  <div class="p-modal__dialog" role="dialog" aria-labelledby="modal-title" aria-describedby="modal-description">
+    <header class="p-modal__header">
+      <h4><i class="p-icon--warning"></i>&nbsp;&nbsp;Disconnecting your repository</h4>
+      <button class="p-modal__close" aria-label="Close active modal" aria-controls="repo-disconnect-modal">Close</button>
+    </header>
+    <p>By disconnecting this repository from your snap, the following will occur:</p>
+    <ul>
+      <li>Previous builds for this snap will disappear from the Builds page</li>
+      <li>You will be able to continue to release your previous revisions on the Releases tab</li>
+    </ul>
+    <p>Do you wish to <strong>continue disconnecting your repository?</strong></p>
+    <form class="p-form" method="POST" action="/{{ snap_name }}/builds/disconnect" name="repoDisconnectForm">
+      <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
+      <p class="u-align--right">
+        <button aria-label="Close active modal" aria-controls="repo-disconnect-modal"class="p-button--neutral">Cancel</button>
+        <button class="p-button--positive p-button-spinner" data-js="repo-disconnect-confirm">
+          <span class="p-button__spinner vanilla-workaround--dark">
```

Comment:
the spinner isn't showing for me

Reply:
honestly I don't know how to fix that. It seems it is related to [the form submit](https://github.com/canonical-web-and-design/snapcraft.io/pull/2654/files#diff-c4e1a01079db29511ae3baf4259a059dR39) - if I don't submit the form, it shows the spinner. I tried to submit the for after 500ms, but it doesn't help... don't know what else to do. Anyway, it works fine on chrome.